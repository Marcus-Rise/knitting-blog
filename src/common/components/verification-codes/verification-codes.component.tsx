import React from "react";
import Head from "next/head";

const VerificationCodes: React.FC<{ yandexCode: string; googleCode: string }> = (props) => {
  return (
    <Head>
      <meta name="google-site-verification" content={props.googleCode} />
      <meta name="yandex-verification" content={props.yandexCode} />
    </Head>
  );
};

export { VerificationCodes };
