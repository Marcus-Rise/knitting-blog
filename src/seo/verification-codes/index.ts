export { VerificationCodes } from "./verification-codes.component";
