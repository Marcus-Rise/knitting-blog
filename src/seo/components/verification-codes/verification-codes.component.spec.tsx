import { render } from "@testing-library/react";
import React from "react";
import { VerificationCodes } from "./verification-codes.component";

describe("VerificationCodes", () => {
  test("render", () => {
    const { asFragment } = render(
      <VerificationCodes googleCode={"google"} yandexCode={"yandex"} />,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
