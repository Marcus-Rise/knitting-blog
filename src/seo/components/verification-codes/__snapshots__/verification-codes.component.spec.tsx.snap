// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerificationCodes render 1`] = `
<DocumentFragment>
  <div
    data-testid="head"
  >
    <meta
      content="google"
      name="google-site-verification"
    />
    <meta
      content="yandex"
      name="yandex-verification"
    />
  </div>
</DocumentFragment>
`;
